source/renderer/app/features/discreet-mode/ui/DiscreetModeFeatureInject.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import type { Node } from 'react';
+import { observer } from 'mobx-react';
+import { useDiscreetModeFeature } from '../context';
+import { DiscreetMode } from '../feature';
+
+type Props = {
+  children: (discreetModeFeature: DiscreetMode) => Node,
+};
+
+const DiscreetModeFeatureInject = ({ children }: Props) => {
```

Comment:
Why is this necessary? I can't find any usage of this component?

Reply:
Removed